job-service/src/main/java/com/hpe/caf/services/job/api/generated/JobsApiServiceImpl.java

```diff
@@ -71,8 +72,12 @@ public Response deleteJobs(final String partitionId, final String jobIdStartsWit
                                final String filter, final String cAFCorrelationId, final SecurityContext securityContext)
             throws Exception
     {
-        final Long successfulDeletions = JobsDelete.deleteJobs(partitionId, jobIdStartsWith, statusType, label, filter);
-        return Response.ok().entity(successfulDeletions).build();
+        final ObjectMapper mapper = new ObjectMapper();
+        final Long value = JobsDelete.deleteJobs(partitionId, jobIdStartsWith, statusType, label, filter);
+        final Map<String, Long> resultMap = Collections.singletonMap("jobsDeleted", value);
+        final String json = mapper.writeValueAsString(resultMap);
```

Comment:
This is weird looking.  Is it definitely returning a JSON object rather than a JSON-encoded string?

Reply:
@dermot-hardy I am now returning an `ObjectNode` built using the Jackson Object Mapper. Think I was possibly just returning a JSON-encoded string before.

Reply:
Sorry, simply removing the `writeValueAsString` function means we are getting JSON object rather than string.